stories/Undo.stories.tsx

```diff
@@ -1,11 +1,36 @@
+import {
+  Meta,
+  Story
+} from '@storybook/react/types-6-0';
```

Comment:
This is probably the right import - https://raw.githubusercontent.com/reaviz/reaviz/master/docs/charts/AreaChart.story.mdx

Reply:
I don't think so, I know I did it because it was the recommend new way of doing it, read about it a few weeks ago. I'll try to take a look to make sure and let you know.

Reply:
https://github.com/storybookjs/storybook/discussions/13931#discussioncomment-375740 the main maintainer is recommending the exact same way of importing types there, 13 days ago.